action.yml

```diff
@@ -43,34 +49,49 @@ runs:
       with:
         repository: ${{ inputs.repository }}
 
+    # Process variables and inputs
+    - id: vars
```

Comment:
should we name it `template_parameters` , just a thought nothing significant

Reply:
Probably not, because it's handling more than just the template.  If we added more parameters they might need to be processed here as well.  Not a bad idea though!  :)